README.md

```diff
@@ -30,6 +30,18 @@ chartjs-plugin-style requires [Chart.js](https://www.chartjs.org). Include Chart
 
 Version 0.3 supports the bevel, drop shadow, inner glow and outer glow effects for datasets and the tooltip. More options are to be added in the upcoming releases.
 
+### Usage in ES6 as module
+
+Nothing else than importing the module should be enough.
+
+```js
+
+...
+import 'chartjs-plugin-style';
+...
```

Comment:
Can you remove the lines of `...` because they are recognized as spread syntax, and I think it's unnecessary.

Reply:
done